src/ctia/events/obj_to_event.clj

```diff
@@ -0,0 +1,61 @@
+(ns ctia.events.obj-to-event
+  (:require [clojure.data :refer [diff]]
+            [schema.core :as s]
+            [ctia.lib.time :as t]
+            [ctia.schemas.actor :refer [StoredActor]]
+            [ctia.schemas.campaign :refer [StoredCampaign]]
+
+            [ctia.events.schemas :as vs]))
+
+
+(s/defn to-create-event :- vs/CreateEvent
+  [model :-  {s/Any s/Any}
```

Comment:
@yogsototh I think you make a good point about using schema here because you want to specify the return type; that is fine.

Here is a what I think about when to use schemas.  It has nothing to do with this code.

We should consider _when_ to use schemas.  (I don't mean to suggest we remove all schemas).  We shouldn't autoamtically use schemas where they don't provide value.  In my opinion, schemas provide value when (1) they are defined on the entry point to an abstraction or, (2) when the readability of the code is noticeably improved (communicating assumptions and providing a contract) or, (2) they are added to improve common test failures by failing faster (higher up the stack).